.chloggen/spanmetricsconnector-independent.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
```

Comment:
Let's remove the changelog entry, it was already added, see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18535/files#diff-35823a4a99fe2d49c1ff990f437d3da0db4646869ac384538ffa9a9d507a236bR8